static/src/js/app/components/DownloadVocab.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <a class="btn btn-sm btn-light" :href="glossesDownload" download="glosses.csv">
-    <icon name="download" /> Export
+    <i class="fa fa-fw fa-download" aria-hidden="true" /> Export
```

Comment:
converting <icon> to <i> might cause reactivity problems

Reply:
I had a good look and I don't think this is true. It looks like the few places that there are interactive icons they are done through css shading. I think we're good.